tidy3d/components/monitor.py

```diff
@@ -552,7 +552,10 @@ class AbstractFieldProjectionMonitor(SurfaceIntegrationMonitor, FreqMonitor):
     far_field_approx: bool = pydantic.Field(
         True,
         title="Far field approximation",
-        description="Whether to enable the far field approximation when projecting fields.",
+        description="Whether to enable the far field approximation when projecting fields. "
+        "If ``True``, terms that decay as O(1/r^2) are ignored, as are the radial components "
+        "of fields. Typically, this should be set to ``True`` only when the projection distance "
+        "is much larger than the size of the device being modeled.",
```

Comment:
out of curiosity, does the wavelength play into whether `far_field_approx` is accurate?

Reply:
Yeah, in that the observations points should be in the far field of the device - I added a sentence to make this explicit. I didn't specify what number of wavelengths would be considered "far field" because I think that can be application-dependent sometimes.